force-app/tests/Shared Code/AccountServiceLayer_Tests.cls

```diff
@@ -59,13 +59,13 @@ private inherited sharing class AccountServiceLayer_Tests {
             false
         );
         Test.stopTest();
-        System.assertEquals(
+        System.Assert.areEqual(
             5,
             updatedAccounts.size(),
             'updatedAccounts should have 5 accounts'
         );
         for (Account ua : updatedAccounts) {
-            System.assert(
+            System.Assert.isTrue(
```

Comment:
Not a deal breaker but this should probably be rewritten with `areEqual` instead of `isTrue`